H/Hwloc/Deps.toml

```diff
@@ -8,3 +8,6 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 
 ["1.0.0"]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+["1.0.2-1"]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
You can have `Test` as a test-only dependency if you want.

Reply:
Done.

Reply:
Done.

Reply:
You need to retrigger the bot in order to update this PR.

Reply:
Done.